Backend/Backend/Infrastructure/FileStorageHelper.cs

```diff
@@ -60,6 +62,26 @@ public enum TranscriptStatus
     };
 
     #region User Profile
+    
+    public static bool ValidateAvatar(IFormFile? file, out object? errorObject) {
+        errorObject = null;
+        if (file is not null) {
```

Comment:
Honestly would recommend just inversing the condition and doing an early return here